modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/StandardHeader.scala

```diff
@@ -48,11 +49,13 @@ case class ObsKeywordReaderImpl(config: Config, telescope: String) extends ObsKe
   override def getGemPrgId: SeqAction[String] = SeqAction(
     config.getItemValue(new ItemKey(OCS_KEY, PROGRAMID_PROP)).toString)
 
-  override def getObsId: SeqAction[String] = {
-    val v = config.getItemValue(new ItemKey(OCS_KEY, OBSERVATIONID_PROP)).toString.split("-").toList.lastOption.getOrElse("")
+  override def getObsId: SeqAction[String] = SeqAction(
+    config.getItemValue(new ItemKey(OCS_KEY, OBSERVATIONID_PROP)).toString
```

Comment:
Hopefully the gem project will give us a more useful type for the observation id